quarkus/tests/integration/src/test/java/org/keycloak/it/storage/map/HotRodStoreDistTest.java

```diff
@@ -32,7 +32,7 @@ public class HotRodStoreDistTest {
     @Launch({ "start", "--optimized", "--http-enabled=true", "--hostname-strict=false" })
     void testSuccessful(LaunchResult result) {
         CLIResult cliResult = (CLIResult) result;
-        cliResult.assertMessage("Experimental feature enabled: map_storage");
+//        cliResult.assertMessage("Experimental feature enabled: map_storage");
```

Comment:
There's no longer CLI output at startup to warn for preview/experimental features.

Reply:
Can we add some debug/trace logging? It should be simpler and only require configuring the `log-level` to the tests.

Reply:
Changed to using a single log statement, and output both for build and start. Doesn't hurt to output for both in case folks are using an optimized build from elsewhere or something they wouldn't necessarily see the build output at all.